__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const app = require("../app");
+const db = require("../db/connection");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const { reduce } = require("../db/data/test-data/articles");
+
+beforeEach(() => seed(data))
+afterAll(() => db.end())
+
+describe("CORE:",() => {
+    describe("Topics:", () => {
+        describe("GET:", () => {
+            it("200: returns all topics", () => {
+                return request(app)
+                .get("/api/topics")
+                .expect(200)
+                .then((response) => {
+                    const {
+                        body: { topics },
+                    } = response;
+                    topics.forEach((topic) => {
```

Comment:
If we're going to iterate through we need to make sure we're doing a length check otherwise we might get a false positive.